components/core/ActivityGroup/components/ProfileInfo.js

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import * as Styles from "~/common/styles";
+
+import { Link } from "~/components/core/Link";
+import { css } from "@emotion/react";
+import { P, H4 } from "~/components/system/components/Typography";
+import { ButtonPrimary, ButtonTertiary } from "~/components/system";
+import { useFollowProfileHandler } from "~/common/hooks";
+
+const STYLES_PROFILE_CONTAINER = css`
+  display: flex;
+  padding-right: 12px;
+  box-sizing: border-box;
+  & > * + * {
+    margin-left: 12px;
+  }
+`;
+
+const STYLES_TEXT_BLACK = (theme) =>
+  css`
+    color: ${theme.system.textBlack};
+    display: inline;
+  `;
+
+const STYLES_PROFILE = css`
+  width: 48px;
+  height: 48px;
+  border-radius: 8px;
+`;
+
+const STYLES_MOBILE_ALIGN = (theme) => css`
+  @media (max-width: ${theme.sizes.mobile}px) {
+    width: 100%;
+    & > span {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
+  }
+`;
+
+export default function ProfileInfo({ owner, viewer, time, action, onAction }) {
+  const { isFollowing, handleFollow } = useFollowProfileHandler({ viewer, user: owner, onAction });
+  const { username, data = {} } = owner;
+  const { photo } = data;
+
+  const isOwner = viewer?.id === owner.id;
+  return (
+    <Link href={`/$/user/${owner.id}`} onAction={onAction}>
+      <div css={STYLES_PROFILE_CONTAINER}>
+        <img src={photo} alt={`${username} profile`} css={STYLES_PROFILE} />
```

Comment:
Instead of this text as alt, can we use some fallback image or just a grey background as a fallback image? I think the text looks really awkward

Could do something like this:
```
<img src="original-image.jpg"
   onerror="this.onerror=null;this.src='default-image.jpg';"
>
```

![image](https://user-images.githubusercontent.com/33686587/125537936-ac6206df-64a0-439b-adbd-10a7fe64716e.png)